code/game/gamemodes/colonialmarines/colonialmarines.dm

```diff
@@ -372,7 +372,21 @@
 				round_statistics.current_map.total_marine_victories++
 				round_statistics.current_map.total_marine_majors++
 		if(MODE_INFESTATION_X_MINOR)
-			musical_track = pick('sound/theme/neutral_melancholy1.ogg','sound/theme/neutral_melancholy2.ogg')
+			var/living_player_list[] = count_humans_and_xenos(EvacuationAuthority.get_affected_zlevels())
+			if(living_player_list[1] && !living_player_list[2]) // If Xeno Minor but Xenos are dead and Humans are alive, see which faction is the last standing
+				var/headcount = count_per_faction()
+				var/living= headcount[5]
```

Comment:
```suggestion
				var/living = headcount[5]
```

Reply:
also accessing by index is rarely ideal, just make the list associated with some defines